keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -96,7 +99,7 @@ class SqlDeliveryConfigRepository(
             name = name,
             application = application,
             serviceAccount = serviceAccount,
-            metadata = metadata ?: emptyMap()
+            metadata = (metadata ?: emptyMap()) + mapOf("createdAt" to ULID.parseULID(uid).timestampAsInstant())
```

Comment:
I decided to store this into `metadata` because I wanted to keep this PR small and refactoring `DeliveryConfig` to expose a new `createdAt` field would be a much larger change that's beyond my goal at the moment (and also not used anywhere besides tests).

Reply:
Fun fact: you can also add a pair to a map (the `mapOf` isn't strictly necessary).